RealmSwift/LinkingObjects.swift

```diff
@@ -220,6 +220,23 @@ public final class LinkingObjects<T: Object>: LinkingObjectsBase {
     /**
      Returns a `Results` containing all the linking objects, but sorted.
 
+     Objects are sorted based on the values of the given key path. For example, to sort a collection of `Student`s from
+     youngest to oldest based on their `age` property, you might call
+     `students.sorted(byKeyPath: "age", ascending: true)`.
+
+     - warning: Collections may only be sorted by properties of boolean, `Date`, `NSDate`, single and double-precision
```

Comment:
It just seems like this warning lists the ways you can sort, this PR adds a way, but it's not reflected in the list. Seems like there ought to be a way to express that here.